README.md

```diff
@@ -171,7 +171,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 - ### **A**
 
   - [Aakansha Priya](https://github.com/priyaaakansha)
-  - [Akhil Rathipelly ](https://github.com/Akhil2707)
+  
```

Comment:
Could remove this extra space, so that we can merge your Pull Request!

Reply:
removed @RakeshSangem

Reply:
Thank you, Akhil